cmd/olm/main.go

```diff
@@ -100,12 +100,11 @@ func main() {
 	http.HandleFunc("/healthz", func(w http.ResponseWriter, r *http.Request) {
 		w.WriteHeader(http.StatusOK)
 	})
-	// TODO: both of the following require vendor updates (add k8s.io/apiserver and update prometheus)
-	//healthz.InstallHandler(mux) //(less code)
-	//mux.Handle("/metrics", promhttp.Handler()) //other form is deprecated
-	http.Handle("/metrics", prometheus.Handler())
 	go http.ListenAndServe(":8080", nil)
```

Comment:
Similar issue here, though this has clearly been here a while

Reply:
See other comments, I'm nearly certain I've confirmed that healthz was being served correctly.